app/views/concepts/_form.html.erb

```diff
@@ -12,10 +12,10 @@
   <% end %>
 
   <div class="field">
-    <%= f.label :name %><br>
-    <%= f.text_field :name %>
+    <%# f.label :name %>
```

Comment:
Rather than commenting out the label, you can add a the Bootstrap class of "sr-only" (screen reader only). This makes the label invisible, but it's still there for accessibility.